Easydict/Swift/View/SettingView/Tabs/TabView/AboutTab.swift

```diff
@@ -27,50 +28,81 @@ struct AboutTab: View {
-                .padding()
-                .shadow(color: .gray, radius: 1, x: 0, y: 0.8)
-
-            VStack(alignment: .leading) {
-                VStack(alignment: .leading) {
-                    Text(appName)
-                        .font(.system(size: 35, weight: .medium))
-                        .padding(.top, 25)
-                        .padding(.bottom, 3)
-
-                    Text("current_version \(version)")
-                        .font(.system(size: 13))
-                        .foregroundColor(.gray)
-
-                    Text(copyrightInfo)
-                        .font(.system(size: 11))
-                        .foregroundColor(.gray)
-                        .padding(.top, 25)
-                        .padding(.bottom, 20)
-                }
-
-                HStack(spacing: 15) {
-                    Button {
-                        NSWorkspace.shared.open(URL(string: "https://github.com/tisfeng/Easydict")!)
-                    } label: {
-                        Label("github_link", systemImage: "star.fill")
-                            .frame(width: 120, height: 20)
+        VortexViewReader { proxy in
+            GeometryReader { geometry in
+                ZStack {
+                    VortexView(.confetti) {
+                        Rectangle()
+                            .fill(.white)
+                            .frame(width: 16, height: 16)
+                            .tag("square")
+
+                        Circle()
+                            .fill(.white)
+                            .frame(width: 16)
+                            .tag("circle")
                     }
-
-                    Button {
-                        NSWorkspace.shared
-                            .open(URL(string: "https://github.com/tisfeng/Easydict/graphs/contributors")!)
-                    } label: {
-                        Label("contributor_link", systemImage: "person.3.sequence.fill")
-                            .frame(width: 120, height: 20)
+                    .frame(height: 220)
+
+                    HStack(alignment: .center, spacing: 30) {
+                        Image("logo") // Make sure to use the correct image name as a string
+                            .resizable()
+                            .renderingMode(.original)
+                            .frame(width: 100, height: 100)
+                            .shadow(color: .gray, radius: 1, x: 0, y: 0.8)
+                            .padding(.bottom, 33)
+                            .padding(.leading, 16)
+                            .padding(.trailing, 16)
+                            .onTapGesture {
+                                proxy.move(
+                                    to: CGPoint(
+                                        // Unify x-point for Settings and Menubar About view
+                                        x: (geometry.size.width / 2) - 159.5,
+                                        y: 70
```

Comment:
Can this coordinate be dynamically calculated rather than hardcoded? 

If we need to modify the UI layout later, will this coordinate need to be adjusted accordingly?

Reply:
The sample code actually provides dynamic calculation
```swift
.onTapGesture { location in
    proxy.move(to: location)
    proxy.burst()
}
```
However, in our case, we're adding confetti to two views with difference window sizes, so I haven't found a solution yet.

https://github.com/tisfeng/Easydict/assets/89069957/ebc71427-2144-49cc-b8b2-1ac6a48ccc32

Reply:
One possible calculation is to find the relationship between the **width & heigh of the window** and **x & y**

We will be able to calculate the location of the logo and tap gesture location based on the window size

Reply:
ok